pkg/controller/registry/reconciler/configmap.go

```diff
@@ -85,6 +85,11 @@ func (s *configMapCatalogSourceDecorator) Service() *v1.Service {
 			Selector: s.Selector(),
 		},
 	}
+
+	labels := map[string]string{}
```

Comment:
Could we use 
```
labels := svc.GetLabels()
if labels == nil {
    labels = map[string]string{}
}
```
That way we don't end up overwriting user-defined labels on services, they may have some set

Reply:
This is a brand-new Service that we create for the first time here so there isn't any user-defined labels. You can see there is no input because this Service is created and managed by OLM.